core/scripts/checkMedic.sqf

```diff
@@ -0,0 +1,43 @@
+
+assignMedicPermission = {
+    private _playerRoleDescription = toLower (roleDescription player);
+    private _AceMedic = "";
+
+
+    if (_playerRoleDescription find "enferm" != -1) then {
+        _AceMedic = 1; // Enfermero
+    } else {
+        if (_playerRoleDescription find "dic" != -1) then {
```

Comment:
When searching for such a short string like **dic**, it is possible that someone will invent a name that includes it. I have seen any type of name in the missions, especially unofficial ones.